src/pages/commits/ActiveCommits/BuildVariantCard.tsx

```diff
@@ -1,20 +1,78 @@
 import styled from "@emotion/styled";
+import IconButton from "@leafygreen-ui/icon-button";
 import { uiColors } from "@leafygreen-ui/palette";
 import { Body } from "@leafygreen-ui/typography";
+import { GroupedTaskStatusBadge } from "components/GroupedTaskStatusBadge";
+import Icon from "components/Icon";
+import { groupStatusesByColor, isFailedTaskStatus } from "utils/statuses";
 
 const { gray } = uiColors;
 
 interface Props {
   buildVariantDisplayName: string;
+  tasks?: {
+    id: string;
+    status: string;
+  }[];
 }
 export const BuildVariantCard: React.FC<Props> = ({
   buildVariantDisplayName,
-}) => <Label key={buildVariantDisplayName}>{buildVariantDisplayName}</Label>;
+  tasks,
+}) => {
+  // get the count of the amount of tasks in each status
+  const { stats } = groupStatusesByColor(
+    tasks.map((task) => ({ ...task, count: 1 }))
+  );
+  const failingTasks = tasks.filter((task) => isFailedTaskStatus(task.status));
+  // get all the umbrellaStatus that are not Failed
+  const otherTasks = stats.filter(
+    (stat) => !isFailedTaskStatus(stat.umbrellaStatus)
+  );
+  return (
+    <>
+      <Label key={buildVariantDisplayName}>{buildVariantDisplayName}</Label>
+      <IconContainer>
+        {otherTasks.map(({ count, umbrellaStatus }) => (
+          <GroupedTaskStatusBadgeWrapper>
+            <GroupedTaskStatusBadge
+              status={umbrellaStatus}
+              key={`${umbrellaStatus}_groupedBadge`}
+              count={count}
+            />
+          </GroupedTaskStatusBadgeWrapper>
+        ))}
+      </IconContainer>
+      <IconContainer>
+        {failingTasks.map(({ id, status }) => (
+          <IconButton
+            aria-label="Failing task icon"
+            onClick={() => {
+              console.log({ id, status });
```

Comment:
Is this a placeholder for upcoming work?

Reply:
Yeah this will be replaced by some analytics as well as linking to a task.